lib/ansible/modules/packaging/os/rhn_register.py

```diff
@@ -286,7 +285,7 @@ def api(self, method, *args):
                 url = "https://%s/rpc/api" % self.hostname
             else:
                 url = "https://xmlrpc.%s/rpc/api" % self.hostname
-            self.server = xmlrpclib.Server(url, verbose=0)
+            self.server = xmlrpc_client(url)
```

Comment:
Am I missing something or is this supposed to be `self.server = xmlrpc_client.ServerProxy(url)`?